tap/passive_tapper.go

```diff
@@ -245,9 +245,10 @@ func startPassiveTapper(opts *TapOpts, outputItems chan *api.OutputChannelItem)
 
 func startTlsTapper(extension *api.Extension, outputItems chan *api.OutputChannelItem, options *api.TrafficFilteringOptions) {
 	tls := tlstapper.TlsTapper{}
-	tlsPerfBufferSize := os.Getpagesize() * 100
+	chunksBufferSize := os.Getpagesize() * 100
```

Comment:
Just a question: Why does the buffer size depends on the page size?

Reply:
Perf and Ring buffers are define in number pages.
It is rounded up to the nearest multiple of the current page size.

There is more info about it here:
https://nakryiko.com/posts/bpf-ringbuf/